.github/ISSUE_TEMPLATE/guidance_request.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Ask a question
+about: Ask for guidance, "how to", or other questions
+labels: question
+---
+
```

Comment:
I would prefer to send the user to the slack channel as we are likely to get more eyes on there

Reply:
> Not sure we need a guidance request but thank you for adding these templates. Some comments for consideration

I could remove it and add the slack channel in our ReadMe file.